plugins/woocommerce-blocks/tests/e2e/tests/all-reviews/all-reviews.block_theme.spec.ts

```diff
@@ -8,55 +8,78 @@ import { expect, test } from '@woocommerce/e2e-playwright-utils';
  */
 import { allReviews } from '../../test-data/data/data';
 
-const blockData = {
-	name: 'woocommerce/all-reviews',
-	selectors: {
-		frontend: {
-			firstReview:
-				'.wc-block-review-list-item__item:first-child .wc-block-review-list-item__text p',
-		},
-	},
-};
-
-test.describe( `${ blockData.name } Block`, () => {
-	test( 'block can be inserted and it successfully renders a review in the editor and the frontend', async ( {
-		admin,
-		editor,
+const BLOCK_NAME = 'woocommerce/all-reviews';
+
+const latestReview = allReviews[ allReviews.length - 1 ];
+
+test.describe( `${ BLOCK_NAME } Block`, () => {
+	test.beforeEach( async ( { admin, editor } ) => {
+		await admin.createNewPost();
+		await editor.insertBlock( { name: BLOCK_NAME } );
+	} );
+
+	test( 'block can be inserted and it sorts reviews by most recent by default', async ( {
+		frontendUtils,
 		page,
 		editorUtils,
 	} ) => {
-		await admin.createNewPost();
-		await editor.insertBlock( { name: blockData.name } );
-
 		await expect( page.getByText( allReviews[ 0 ].review ) ).toBeVisible();
 
 		await editorUtils.publishAndVisitPost();
 
-		await expect( page.getByText( allReviews[ 0 ].review ) ).toBeVisible();
+		const block = await frontendUtils.getBlockByName( BLOCK_NAME );
+		const reviews = block.locator(
+			'.wc-block-components-review-list-item__text'
+		);
+
+		await expect( reviews.nth( 0 ) ).toHaveText( latestReview.review );
 	} );
 
-	test( 'can change sort order in the frontend', async ( {
+	test( 'can sort by highest rating in the frontend', async ( {
 		page,
 		frontendUtils,
+		editorUtils,
 	} ) => {
-		await page.goto( '/all-reviews-block/' );
+		await editorUtils.publishAndVisitPost();
 
-		const block = await frontendUtils.getBlockByName( blockData.name );
-		let firstReview;
-		firstReview = block.locator( blockData.selectors.frontend.firstReview );
+		const block = await frontendUtils.getBlockByName( BLOCK_NAME );
+		const reviews = block.locator(
+			'.wc-block-components-review-list-item__text'
+		);
 
-		// The most recent review should be at top, TODO: this assertion could be improved.
-		await expect( firstReview ).toHaveText( allReviews[ 2 ].review );
+		await expect( reviews.nth( 0 ) ).toHaveText( latestReview.review );
```

Comment:
nit: `nth( 0 )` can be replaced with `first()`

Reply:
Thanks thats nicer! Fixed https://github.com/woocommerce/woocommerce/pull/44880/commits/40cabeabe5fd2f2a3f4bd338ace15715fe69d726